components/Wrapped/slides/PlatinumCardSlide.tsx

```diff
@@ -0,0 +1,100 @@
+import $ from "@/utils/theme";
+import type { SlideProps, SlideOptions } from "../internals/slidesHelper";
+import Background from "../components/Background";
+import styles from "@/styles/PlatinumCardSlide.module.css";
+
+export default function PlatinumCardSlide({ data }: SlideProps) {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        flexGrow: 1,
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100%"
+      }}
+    >
+      <h1
+        {...$.title({
+          // You can apply classes by daisy-chaining them after the $
+          // You can then add CSS properties like you
+          // normally would inside of this object
+          color: "white",
+          alignSelf: "start",
+          justifySelf: "flex-start"
+        })}
+      >
+        April Fools!
```

Comment:
Let's make this copy something a little more comprehensive. 

```suggestion
        On the first day of April, you  earned yourself a one-of-a-kind exclusive HCB Platinum Card. April Fools? 
```

Thoughts on that? @YodaLightsabr got any ideas?

Reply:
That sounds good

Reply:
cool!